docs/2-Collection_Containers.adoc

```diff
@@ -965,6 +965,64 @@ They are named *{IntHashSet}*, *{FloatHashSet}* etc.
 *{BooleanHashSet}* is implemented using a single integer to hold one of four possible states: ([], [F], [T], or [T, F]).
 All other sets use open addressing and quadratic probing.
 
+There are different ways you can create *IntHashSet*, which are listed as below.
+You can create an primitive sets using **newSetWith** method. You can initialize an *IntHashSet*  using constructor argument. There are 5 different overloaded constructor are available.
+Below examples are provided with *IntHashSet* but all are same for *FloatHashSet* and *BooleanHashSet* as well.
+[source,java]
+----
+// Initialize IntHashSet using newSetWith method, which is taking int vargs.
+IntHashSet intHashSet1 = IntHashSet.newSetWith(1, 2, 3, 4, 5);
+FloatHashSet floatHashSet1 = FloatHashSet.newSetWith(12.5f, 13.5f, 14.88f, 15.6f);
+BooleanHashSet booleanHashSet = BooleanHashSet.newSetWith(true, false);
+
+// Empty Constructor will initialize IntHashSet with 16 size
+IntHashSet intHashSet2 = new IntHashSet();
+
+// Initialize Constructor with initialCapacity
+IntHashSet intHashSet3 = new IntHashSet(32);
+
+// Initialize Constructor with set of integers using vargs.
+IntHashSet intHashSet4 = new IntHashSet(1, 2, 3, 4, 5, 6);
+
+// Initialize Constructor with IntIterable.
+IntIterable intIterable = intHashSet4; // you can get from any list, set which implement this interface
+IntHashSet intHashSet = new IntHashSet(intIterable);
```

Comment:
Change the type on the left to the interface `MutableIntSet`.

Reply:
Thank you for your suggestion. I have implemented the suggested comment.